tests/Fixer/PhpTag/NoClosingTagFixerTest.php

```diff
@@ -121,9 +121,12 @@ function bar()
                 '<?php ?>',
             ),
             'No code, only comment' => array(
-                '<?php /* license */ ',
+                '<?php /* license */',
```

Comment:
hmm not sure why this was done in the first place, it seems to be doing that for all cases